bpf/lib/ipv4.h

```diff
@@ -14,21 +14,20 @@ struct ipv4_frag_id {
 	__be16	id;		/* L4 datagram identifier */
 	__u8	proto;
 	__u8	pad;
-} __attribute__((packed));
+} __packed;
 
 struct ipv4_frag_l4ports {
 	__be16	sport;
 	__be16	dport;
-} __attribute__((packed));
+} __packed;
 
-#if defined IPV4_FRAGMENTS
+#ifdef ENABLE_IPV4_FRAGMENTS
 struct bpf_elf_map __section_maps IPV4_FRAG_DATAGRAMS_MAP = {
 	.type           = BPF_MAP_TYPE_LRU_HASH,
 	.size_key	= sizeof(struct ipv4_frag_id),
 	.size_value	= sizeof(struct ipv4_frag_l4ports),
 	.pinning	= PIN_GLOBAL_NS,
 	.max_elem	= CILIUM_IPV4_FRAG_MAP_MAX_ENTRIES,
-	.flags		= CONDITIONAL_PREALLOC,
```

Comment:
Could this also be the fix for #10925?

Reply:
Seems likely. The `-ENOTSUPP` returned would be consistent.